.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy site
+
+on:
+  push:
+    branches:
+      - 'master'
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy site to AWS S3
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: ${{ secrets.AWS_REGION }}
+        role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+
+    - name: Build
+      run: mkdir dist/ && cp -a img/ js/ style/ index.html dist/ && test -d dist/
```

Comment:
I guess following-up, usually SSGs will have a separate dir and build the site and deploy everything from the "built" dir, kinda like you are doing with "dist" here. But might be easier to do that in a script. Even if not I would recommend a subfolder for all the site content simply for the reason I stated above and it would avoid copying e.g. the `.git` and `.github` folders from the top-level, but also would prevent forgetting to add any new files here in this script.

Reply:
Sure, keeping it in a subdir makes sense. 

I don't anticipate this being a complex site with more than one page, it will basically be the same as the old Dash Electrum page. That was built the same way, the main change here is I'm attempting to use an AWS role in CI. So no need to plan for SSGs I think, we can make those changes and add a build script in a separate PR in the future if necessary.